index.php

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+       
+  <title><?php echo "Assignment 6"; ?></title>
+  <meta charset="utf-8"> 
+</head>
+<body>
+<h1><?php echo "Hello, word!"; ?></h1>
+<div>Allow me to introduce myself ! My name is <?php echo "Mahoug Houffon"; ?></div>
```

Comment:
Good php statements! Complete with quotation marks and that necessary semicolon! 👍